stdlib/3/typing.pyi

```diff
@@ -95,6 +95,8 @@ class SupportsAbs(Protocol[_T_co]):
 
 @runtime
 class SupportsRound(Protocol[_T_co]):
+    @abstractmethod
+    def __round__(self) -> _T_co: ...
```

Comment:
Wait, why is this needed? `ndigits` is already an optional argument.

Reply:
The first version of this PR was busted and only passed testing because I introduced errors that allowed it to pass <_<.

In the just-pushed updated version, the overload is fixed to properly specialize the type behavior, which is that it should return `int` when called without ndigits. (This matches the signature of `round` and the signatures of `__round__` on concrete types.)